python/ccxt/bingx.py

```diff
@@ -1124,19 +1124,13 @@ def fetch_balance(self, params={}):
         query for balance and get the amount of funds available for trading or funds locked in orders
         see https://bingx-api.github.io/docs/#/spot/trade-api.html#Query%20Assets
         see https://bingx-api.github.io/docs/#/swapV2/account-api.html#Get%20Perpetual%20Swap%20Account%20Asset%20Information
-        see https://bingx-api.github.io/docs/#/standard/contract-interface.html#Query%20standard%20contract%20balance
```

Comment:
@ZetZ1 can you please try to revert this file? We should only edit the TS one which I already did but for some reason can't restore this file.